Dockerfile

```
@@ -0,0 +1,5 @@
+FROM 3.9-alpine
+
+COPY . /app
+WORKDIR /app
+ENTRYPOINT ["/entrypoint.sh"]
```

it's best to leave trailing newlines 
```suggestion
ENTRYPOINT ["/entrypoint.sh"]

```